styles.css

```diff
@@ -0,0 +1,785 @@
+/* Color Pallete
+
+#272a31
+#ec5242
+#d3d3d3
+
+*/
+
+/* General Styles
+************************** */
+
+@font-face {
+  font-family: 'cocogoose';
+  src: url('fonts/Cocogoose.otf');
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+  font-size: 3vw;
+  color: #272a31;
+}
+
+.fa-bars,
+.fa-x {
+  width: 8vw;
+}
+
+.fa-bars {
+  margin: 4vw 0 3vw 4vw;
+}
+
+hr {
+  border: 1px solid #ec5242;
+  width: 14vw;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.fb {
+  width: 16px;
+}
+
+.tw {
+  width: 25px;
+}
+
+/* Header Styles
+************************** */
+
+.hidden {
+  display: none;
+}
+
+.mobile-view {
+  position: absolute;
+  left: 0;
+  top: 0;
+  background: #272a31;
+  opacity: 0.9;
+  width: 100vw;
+  height: 100vh;
+  backdrop-filter: blur(5px);
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 6vw;
+  filter: blur(40);
+  z-index: 1;
+}
+
+.footer-content li {
+  margin-bottom: 5px;
+}
+
+.mobile-view .navbar-1 {
+  display: none;
+}
+
+.mobile-view .navbar-2 {
+  order: 2;
+  text-align: center;
+}
+
+.mobile-view .navbar-2 li {
+  padding: 10px;
+  margin: 10px;
+}
+
+.mobile-view .fa-x {
+  order: 1;
+  margin: 10px 0 30px 70vw;
+}
+
+.mobile-view .fb {
+  width: 25px;
+}
+
+.mobile-view .tw {
+  width: 35px;
+}
+
+.mobile-view .logo {
+  width: 50vw;
+}
+
+.mobile-view a {
+  color: #d3d3d3;
+  text-decoration: none;
+}
+
+/* Footer Styles
+************************** */
+
+.footer {
+  padding: 40px 0 40px;
+}
+
+.home-footer-container {
+  display: flex;
+}
+
+.footer img {
+  width: 40vw;
+  margin: 5vw 5vw;
+}
+
+.footer-content {
+  width: 45vw;
+}
+
+.footer-content a {
+  font-size: 2.5vw;
+  text-decoration: none;
+}
+
+.agenda a {
+  display: inline-block;
+  color: #f2f1f1;
+  text-decoration: none;
+  font-size: 4.5vw;
+  text-align: center;
+  background-color: #ec5242;
+  padding: 4vw;
+  width: 100%;
+}
+
+.home.footer-content a {
+  color: #272a31;
+}
+
+/* About Footer Section Styles
+*********************************/
+
+.footer.about {
+  background: #272a31;
+  color: #f2f1f1;
+}
```

Comment:
- Please let's use the original background color according to the design which is #2E2E2E